src/main/kotlin/uk/gov/justice/digital/hmpps/prisonercontactregistry/service/PrisonerContactRegistryService.kt

```diff
@@ -45,10 +51,31 @@ class PrisonerContactRegistryService(private val prisonApiClient: PrisonApiClien
     return contacts
   }
 
+  fun getApprovedSocialContactList(
+    prisonerId: String,
+    personId: Long? = null,
+    withAddress: Boolean,
+    hasDateOfBirth: Boolean? = null,
+    notBannedBeforeDate: LocalDate? = null,
+  ): List<ContactDto> {
+    log.debug("getContactList called with parameters : prisonerId - {}, personId - {}, withAddress - {}, hasDateOfBirth - {}, notBannedBeforeDate - {}", prisonerId, personId, withAddress, hasDateOfBirth, notBannedBeforeDate)
```

Comment:
should the log line be:

"getApprovedSocialContactList called with parameters : ......"

Reply:
Updated